libdirections-hybrid/gradle.properties

```diff
@@ -1,3 +1,4 @@
 POM_ARTIFACT_ID=router
+POM_VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
Could we do something different so we don't need to change the version name to all of the core modules if we want to cut a release?